SAMtags.tex

```diff
@@ -187,17 +189,17 @@ \subsection{Additional Template and Mapping data}
 the reference; the deleted sequence is AC; the last 6~bases are matches.
 The {\tt MD} field ought to match the {\sf CIGAR} string.
 
-\item[MQ:i:\tagvalue{}]
+\item[MQ:i:\tagvalue{score}]
 Mapping quality of the mate/next segment.
 
-\item[NH:i:\tagvalue{}]
-Number of reported alignments that contains the query in the current record.
+\item[NH:i:\tagvalue{count}]
+Number of reported alignments that contain the query in the current record.
 
 \item[NM:i:\tagvalue{}]
```

Comment:
distance?

Reply:
This PR doesn't touch NM. I'd leave that to #308…

Reply:
I avoided "distance" because frankly it gets into the whole world of preconceived ideas as to what an edit distance is.  I figured "count" was sufficient for NM as it's used a lot elsewhere and the actual description for what is being counted follows.